server/src/main/java/com/linecorp/centraldogma/server/internal/admin/service/RepositoryService.java

```diff
@@ -177,10 +177,11 @@ public CompletionStage<List<CommitDto>> getHistory(@Param String projectName,
                                                        @Param String path,
                                                        @Param @Default("-1") String from,
                                                        @Param @Default("1") String to) {
+        final String pathPattern = "/".equals(path) ? "/**" : path + "/**";
```

Comment:
The other APIs we have (e.g. watch*, diff*) don't add a `slash` at the end.
For consistency, I think it would be better that all APIs that accept a `pathPattern` have the same behavior.
Also, I think it is perfectly valid to watch `/a**` depending on the use-case.

Question) What do you think of fixing this issue from the client side instead?

Reply:
Hi @jrhee17, about fixing this issue from the client side, do you mean we receive the list result from the client and filter?
From my side, it is not good to do logic on the client side. 
@minwoox How do you think?

Reply:
Sorry about the late reply. Yeah, I also thought we should fix the UI (which is the client) to call the path correctly when I first created the issue.
We should let users retrieve whatever history they want so I think we should touch it on the server-side.

Reply:
Sound good @minwoox, could you provide me the screenshot to explain expected from client? I am happy to finish this 🚀